keda/values.yaml

```diff
@@ -39,12 +49,6 @@ podLabels:
   keda: {}
   metricsAdapter: {}
 
-## See `kubectl explain poddisruptionbudget.spec` for more
-## ref: https://kubernetes.io/docs/tasks/run-application/configure-pdb/
-podDisruptionBudget: {}
```

Comment:
This is a breaking change, can you make it non-breaking please?

Reply:
Any hint how i could do that without breaking?